client/signup/steps/website-content/section-types/default-page-details.tsx

```diff
@@ -134,7 +150,21 @@ export function DefaultPageDetails( {
 				value={ page.content }
 				error={ formErrors[ fieldName ] }
 				label={ description }
+				disabled={ !! page.useFillerContent }
+				hasFillerContentCheckbox={ isEnglishLocale }
```

Comment:
I'm wondering if this is the culprit of the following warning:
<img width="537" alt="image" src="https://user-images.githubusercontent.com/1842898/224884724-d1f04741-981a-4210-8714-211ddb54aa31.png">

Reply:
Good catch! The new prop was being forwarded to both the fieldset and textarea elements. I fixed it via https://github.com/Automattic/wp-calypso/pull/73859/commits/9533a351afada1b89d8ad53547de68633df6efff.